### PR TITLE
Add deployment ID as url link

### PR DESCRIPTION
**Pull request by Komorebi-E**
## Description

Constructs a URL link based on the deployment ID. It's available in all of the codedeploy messages.

## Related Issue

[#304](https://github.com/skyscrapers/sweepbright/issues/304)

## Motivation and Context

Customer requested that there be a link to the deployment ID.

## How Has This Been Tested?

Tested with test lambda account and the requesting customer's account to make sure formatting of the url works.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: notify-slack/functions/lambda-slack.py [hunk 1]
```diff
@@ -34,8 +35,10 @@ def send_slack(message):
     icon_emoji = ":codedeploy:"
     title = message['status']
     pretext = ""
-    text = "The deployment for app *%s* in group %s\n with id `%s`" % ( message['applicationName'], 
-    message['deploymentGroupName'], message['deploymentId'])
+    deployment_url = '<https://' + region + '.console.aws.amazon.com/codesuite/codedeploy/deployments/' + message['deploymentId'] + '?region=eu-west-1|' + message['deploymentId'] + '>'
```
Comment (SamClinckspoor) [inline notify-slack/functions/lambda-slack.py]: `?region=eu-west-1` should be `?region=' + region + '`

**Review by SamClinckspoor — CHANGES_REQUESTED** [review]
small comment

File: notify-slack/functions/lambda-slack.py (hunk 1 above)
Comment (Komorebi-E) [reply notify-slack/functions/lambda-slack.py]: Fixed, it's now `'?region=' + region + '|' + message['deploymentId'] + '>'`

**Review by SamClinckspoor — APPROVED** [review]
Not sure if the zip should be in the repo.

**Review by duboisph — APPROVED** [review]

**Pull request merged**